README.md

```diff
@@ -1,24 +1,225 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+<div align="center">
+  <br/>
+  <br/>
+  <br/>
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="./app/assets/images/logo3.svg" alt="logo" width="400"  height="auto" />
+  <br/>
+  <br/>
+  <br/>
+  <h1><b>Blog App</b></h1><a name="about-project"></a>
+</div>
 
-Things you may want to cover:
 
-* Ruby version
 
-* System dependencies
 
-* Configuration
 
-* Database creation
 
-* Database initialization
+<!-- TABLE OF CONTENTS -->
 
-* How to run the test suite
+# 📗 Table of Contents
 
-* Services (job queues, cache servers, search engines, etc.)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Blog App ](#-about-project)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
 
-* Deployment instructions
+<!-- PROJECT DESCRIPTION -->
 
-* ...
+# 📖 Blog App <a name="about-project"></a>
+
+> This is an Blog app which allows you to show the list of posts and empower readers to interact with them by adding comments and liking posts.
```

Comment:
- [Optional] Hi @SergioPeralta22, I think the beginning of the statement is not clear. What do you think of saying "This is **a** Blog app"?.